test/parallel/test-process-title.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+const common = require('../common');
+const { spawnSync } = require('child_process');
+const { strictEqual } = require('assert');
+
+// FIXME add sunos support
+if (common.isSunOS)
+  common.skip(`Unsupported platform [${process.platform}]`);
+// FIXME add IBMi support
+if (common.isIBMi)
+  common.skip('Unsupported platform IBMi');
+
+// Explicitly assigning to process.title before starting the child process
+// is necessary otherwise *its* process.title is whatever the last
+// SetConsoleTitle() call in our process tree set it to.
+if (common.isWindows)
+  process.title = process.execPath;
```

Comment:
Using `// Flags: --title=test` to explicitly set the title with a flag would here also, right? Might be better than having a special case here for Windows

Reply:
See https://github.com/libuv/libuv/issues/2667 - I believe this is a Windows-only libuv bug. Until it's fixed, it's best to call it out explicitly.